plugins/block-plus-minus/package.json

```diff
@@ -38,8 +38,8 @@
         "src"
     ],
     "devDependencies": {
-        "@blockly/dev-scripts": "^0.20200403.0",
-        "@blockly/dev-tools": "^0.20200313.1",
+        "@blockly/dev-scripts": "^0.20200403.1",
```

Comment:
FYI, we should be holding off on publishing this one. due to https://github.com/google/blockly-samples/issues/219

Reply:
Unfortunately, too late. This gets generated as part of the publish script.